deploy/helm/minikube-dev.yml

```diff
@@ -39,6 +39,22 @@ web-server:
   fallbackUrl: "https://data.gov.au"
   featureFlags:
     cataloguing: true
+  dateConfig:
```

Comment:
Not a big deal, but you should be able to leave these out if you want, it'll fall back to the values in `values.yaml`

Reply:
Nope, good catch!